coderedcms/templates/coderedcms/includes/iframe_gmap.html

```diff
@@ -1,7 +1,7 @@
 {% if address %}
 <div class="my-2">
     <div class="embed-responsive embed-responsive-16by9 mb-5">
-        <iframe class="embed-responsive-item" width="100%" style="border:0" src="https://maps.google.com/maps?q={{ address }}&output=embed" allowfullscreen></iframe>
+        <iframe class="embed-responsive-item" width="100%" style="border:0" src="https://maps.google.com/maps?q={{ address }}&output=embed" title="Map of {{address}}" allowfullscreen></iframe>
```

Comment:
Just noticed this... the "Map of" text should be marked translatable using `{% trans %}` tag. Something like: `{% trans 'Map of' %} {{ address }}` should do the trick I think. There are similar examples in other templates.